kafka_utils/util/zookeeper.py

```diff
@@ -134,9 +141,29 @@ def set_topic_config(self, topic, value):
                 config_data
             )
             # Create change
+            version = kafka_version[1]
+
+            # this feature is supported in kafka 9 and kafka 10
+            assert version in [9, 10], "Feature supported with kafka 9 and kafka 10"
```

Comment:
Let's use a set here: {9, 10}. It's doesn't make any difference in this case, but it's a bit nicer.